src/toggle/types.ts

```diff
@@ -21,8 +21,9 @@ export interface ToggleProps
     styleType?: ToggleStyleType | undefined;
     /** The Toggle label */
     children: React.ReactNode;
+    childrenMaxLines?: { mobile?: number; desktop?: number } | undefined;
```

Comment:
what is this used for?

Reply:
this is to set the maxLines to show for title.